### PR TITLE
Change spelling of CDT.cloud to CDT Cloud

### PR DESCRIPTION
**Pull request by planger**

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Notices for Eclipse CDT.cloud
+# Notices for Eclipse CDT Cloud
 
-This content is produced and maintained by the Eclipse CDT.cloud project.
+This content is produced and maintained by the Eclipse CDT Cloud project.
 
-* Project home: https://projects.eclipse.org/projects/ecd.cdt.cloud
+* Project home: https://projects.eclipse.org/projects/ecd.CDT Cloud
```
Comment (sdirix) [inline NOTICE.md:5]:
```suggestion
* Project home: https://projects.eclipse.org/projects/ecd.cdt-cloud
```

**Review by sdirix — CHANGES_REQUESTED** [review]
Found one small issue

**Comment by planger** [comment]
Thank you, I committed your suggestion!

**Review by sdirix — APPROVED** [review]
LGTM

**Pull request merged**